From 9cda32c0af1651ea84f45d13da285fc732c95ab9 Mon Sep 17 00:00:00 2001
From: Viorel Suman <viorel.suman@nxp.com>
Date: Wed, 18 Jul 2018 11:03:19 +0300
Subject: [PATCH 4213/5242] MLK-18955-1: ARM64: dts: imx8mm: fix SPDIF1 core
 and system clocks

commit  abbb423d25098863eb45edb8a2a8db3f89a72ae8 from
https://source.codeaurora.org/external/imx/linux-imx.git

The proper SPDIF1 core and system clocks are AUDIO AHB clock.

Signed-off-by: Viorel Suman <viorel.suman@nxp.com>
Reviewed-by: Shengjiu Wang <shengjiu.wang@nxp.com>
(cherry picked from commit 5b8537746c2eb5ede8216e104ae75c38b82b7ce0)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
index 32e6a63..9f962ad 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
@@ -783,13 +783,13 @@
 		compatible = "fsl,imx8mm-spdif";
 		reg = <0x0 0x30090000 0x0 0x10000>;
 		interrupts = <GIC_SPI 6 IRQ_TYPE_LEVEL_HIGH>;
-		clocks = <&clk IMX8MM_CLK_IPG_AUDIO_ROOT>, /* core */
+		clocks = <&clk IMX8MM_CLK_AUDIO_AHB_DIV>, /* core */
 			 <&clk IMX8MM_CLK_24M>, /* rxtx0 */
 			 <&clk IMX8MM_CLK_SPDIF1_DIV>, /* rxtx1 */
 			 <&clk IMX8MM_CLK_DUMMY>, /* rxtx2 */
 			 <&clk IMX8MM_CLK_DUMMY>, /* rxtx3 */
 			 <&clk IMX8MM_CLK_DUMMY>, /* rxtx4 */
-			 <&clk IMX8MM_CLK_IPG_AUDIO_ROOT>, /* rxtx5 */
+			 <&clk IMX8MM_CLK_AUDIO_AHB_DIV>, /* rxtx5 */
 			 <&clk IMX8MM_CLK_DUMMY>, /* rxtx6 */
 			 <&clk IMX8MM_CLK_DUMMY>, /* rxtx7 */
 			 <&clk IMX8MM_CLK_DUMMY>; /* spba */
-- 
1.7.9.5

